e2e/testcases/invalid_auth_test.go

```diff
@@ -55,7 +55,9 @@ func TestInvalidAuth(t *testing.T) {
 	if err != nil {
 		nt.T.Fatal(err)
 	}
-	commitHash := nt.RootRepos[configsync.RootSyncName].MustHash(nt.T)
+	// TODO: What's the commit in the rsync status if the repo doesn't exist and/or auth fails?
```

Comment:
This is a good catch, do we have an issue to track this?

Reply:
Filed b/361182373

Reply:
Updated the TODOs